### PR TITLE
Feat: unified nav bar

### PR DESCRIPTION
**Pull request by sorodrigo**
- Updates styles of the nav bar.
- Updates styles of explore page to match visuals.

### CODE REVIEW

File: frontend/styles/_settings.scss [hunk 1]
```diff
@@ -80,6 +80,8 @@ $light-gray: #f6f6f6;
 $mid-light-gray: #e5e5e5;
 $medium-gray: #dddddd;
 $gray-text: #515253;
+$yellow-gray: #d9d9d9;
+$yet-another-gray: #999999;
```
Comment (Bluesmile82) [inline frontend/styles/_settings.scss]: 😆😆😆 
Maybe something more descriptive? Although we have a lot it's true. Is the only one we will remember

**Comment by Bluesmile82** [comment]
Could you change the text in the logo for the Svg that we are supposed to have? The font looks different:

Currently:


Design:

**Comment by Bluesmile82** [comment]
Could we reduce a bit the padding of the text on the cards so the texts shows a bit higher:

Current:
image

Design:
image

File: frontend/styles/_settings.scss (hunk 1 above)
Comment (sorodrigo) [reply frontend/styles/_settings.scss]: I'm already at the point where I need to search which gray in the design equals which variable. I'm not sure it's worth it

**Comment by sorodrigo** [comment]
> Could we reduce a bit the padding of the text on the cards so the texts shows a bit higher
You're comparing mobile vs desktop, this is how it actually looks like in desktop:
Screen Shot 2020-01-09 at 08 53 21

I did realise that the separator between country and commodity is larger and the font of the subtitle is bold, will make those changes

**Pull request merged**